drivers/media/i2c/imx219.c

```diff
@@ -112,6 +112,15 @@
 #define IMX219_TESTP_BLUE_DEFAULT	0
 #define IMX219_TESTP_GREENB_DEFAULT	0
 
+/* Embedded metadata stream structure */
+#define IMX219_EMBEDDED_LINE_WIDTH 16384
+#define IMX219_NUM_EMBEDDED_LINES 1
+
+enum pad_types {
+	IMAGE_PAD,
+	METADATA_PAD,
+};
```

Comment:
You could add a count/num terminator (I'd call it PAD_COUNT if the names were the other way round) and use it in place of the literal `2`s below. Otherwise it looks reasonable to the untrained eye.

Reply:
Will fix.

Reply:
If you have a PAD_COUNT, then all the ```if (code->pad > METADATA_PAD)``` checks ought to be ```if (code->pad >= PAD_COUNT```.

Reply:
Done.